packages/editor-ui/src/stores/templates.store.ts

```diff
@@ -131,8 +131,10 @@ export const useTemplatesStore = defineStore(STORES.TEMPLATES, {
 				utm_n8n_version: rootStore.versionCli,
 				utm_awc: String(workflowsStore.activeWorkflows.length),
 			};
-			if (userStore.currentUserCloudInfo?.role) {
-				defaultParameters.utm_user_role = userStore.currentUserCloudInfo.role;
+			const userRole: string | undefined =
+				userStore.currentUserCloudInfo?.role ?? userStore.currentUser?.personalizationAnswers?.role;
```

Comment:
Are the values the same in both cases? Should we normalize them here or prepare for both value in the templates site?

Reply:
Nope, they are not connected in any way and the website has a setting to map different survey roles to template category